packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/services/workspace-metadata-updater.service.ts

```diff
@@ -145,6 +135,15 @@ export class WorkspaceMetadataUpdaterService {
       'workspaceId',
     ]);
 
+    /**
+     * Create field metadata
+     */
+    const createdFieldMetadataCollection = await fieldMetadataRepository.save(
```

Comment:
Inverted create and update because when deprecating a field and creating a new one with the deprecated field's former name, we first need the deprecated field to be renamed before we can create a new one with the previous name